Server/Utilities/SpringBoardAlerts.m

```diff
@@ -124,9 +124,7 @@ - (instancetype)init_private {
 // Some language names can be different in different Xcode versions
 // function maps language names to make sure that all xcode versions will work
 - (NSString *)fixLanguageName:(NSString *)languageName {
-    if (![languageName isEqualToString: @"en-US"]) {
```

Comment:
We currently don't have a en_US.json with springboard localizations.
Maybe it's better to remove "!" symbol in the condition?
Also there is the line 112 in this file which maps any non "en" localization to "en".